engine/js/BertaEditor.js

```diff
@@ -218,6 +218,18 @@ var BertaEditor = new Class({
               }
             }
 
+            // Entry moving to other section
+            $$('.js-bt-open-move-entry-to-section').addEvent('click', function (e) {
```

Comment:
I would suggest using plain JS here, so it would be easier to remove moo tools in the future.